ietf/doc/models.py

```diff
@@ -702,7 +702,7 @@ def is_downref(self):
             else:
                 target_lvl = self.target.document.intended_std_level.slug
 
-        rank = { 'ps':1, 'ds':2, 'std':3, 'bcp':3 }
+        rank = { 'ps':1, 'ds':2, 'std':3, 'bcp':1 }
```

Comment:
This is not doing what you intend.

Add tests to prove that the result is what is intended.

I'd look more closely at the conditional to see if we have a logic inversion there.